### DISCUSSION TITLE
How to Regenerate / Rebuild default Fleet Agent Polices

### DISCUSSION THREAD

**Discussion by oneCrazyAdmin** [Security-Onion-Solutions/securityonion#12492] [2.4]
### Version

2.4.50

### Installation Method

Security Onion ISO image

### Description

other (please provide detail below)

### Installation Type

Distributed

### Location

on-prem with Internet access

### Hardware Specs

Exceeds minimum requirements

### CPU

120

### RAM

320Gb

### Storage for /

100Gb

### Storage for /nsm

1.5Tb per node

### Network Traffic Collection

tap

### Network Traffic Speeds

more than 10Gbps

### Status

Yes, all services on all nodes are running OK

### Salt Status

No, there are no failures

### Logs

Yes, there are additional clues in /opt/so/log/ (please provide detail below)

### Detail

There was an issue to where we had to reset the fleet server because it was not working correctly.  This was in part because of an elastic cluster failure.  

Now the Fleet server is up and working, but in the process all of the Fleet Agent Policies that are default are gone.  Is there a way to regenerate those?

### Guidelines

- [X] I have read the discussion guidelines at https://github.com/Security-Onion-Solutions/securityonion/discussions/1720 and assert that I have followed the guidelines.

**Comment by cm-ops** [comment]
Check for `/opt/so/state/eaintegrations.txt`, if it is present, remove it and run `so-elastic-fleet-integration-policy-load`.
